packages/omgidl-parser/src/parseIDL.test.ts

```diff
@@ -1538,7 +1580,7 @@ module rosidl_parser {
             type: "uint32",
             isComplex: false,
             isConstant: true,
-            value: 0,
+            value: 51,
```

Comment:
did we make up this COLORS example ourselves or did we get it from somewhere else? Just wondering why we didn't notice this issue before

Reply:
yeah I made it according to our previous behavior.